lib/sentry/opentelemetry/span_storage.ex

```diff
@@ -1,78 +1,81 @@
 defmodule Sentry.Opentelemetry.SpanStorage do
   @moduledoc false
+  use GenServer
 
-  @root_spans_table :sentry_root_spans
-  @child_spans_table :sentry_child_spans
+  @table :span_storage
```

Comment:
Scope this to Sentry, as the ETS namespace is global. My suggestion is

```suggestion
  @table __MODULE__
```

Reply:
In one of our pairing sessions for `check_in_id_mappings.ex`, we discussed this being bad practice to have both the GenServer and the table name the same. Thoughts?